Add session id requirements to docs (#9386)
PostHog/posthog.com: contents/docs/data/sessions.mdx

    alt="Entry URL breakdown" 
    classes="rounded"
/>

> 
> ### Custom session IDs

Reviewer: ```suggestion

## Custom session IDs
```

Our server-side SDKs do not send `$session_id` by default. If you want to use sessions with server-side events, you can set a custom `$session_id` property. This will let you use session aggregations for these events, and can be useful if you want to connect server-side events to session replays or web analytics sessions.

Custom session IDs must meet the following requirements:
* Must be consistent across events in the same session

> * Must not be reused by a different user or for a different session

Reviewer: ```suggestion
* Must not be reused by a different user or session
```

* Must be a valid UUIDv7
* The timestamp part of the UUIDv7 must be equal to or before the timestamp of the first event in the session
* The timestamp part of the UUIDv7 plus 24 hours must be after the timestamp of the last event in the session

> Any events that do not meet these requirements will not be included in session aggregations.

Reviewer: ```suggestion
Any events with session IDs that do not meet these requirements are not included in session aggregations.
```

> Our [JavaScript Web library](/docs/libraries/js) and mobile SDKs (Android, iOS, React Native, and Flutter) will automatically create session IDs for you, but if you override them, you must ensure they meet these requirements.

Reviewer: ```suggestion
Our [JavaScript Web library](/docs/libraries/js) and mobile SDKs (Android, iOS, React Native, and Flutter) automatically create session IDs for you, but if you override them, you also must ensure they meet these requirements.
```
